src/pages/profile/[user_id].tsx

```diff
@@ -1,33 +1,54 @@
 import { Suspense, useContext, useEffect, useState } from "react";
-
 import { NextRouter, useRouter } from "next/router";
 
 import { userContext } from "supabase/user_context";
 import { useSupabaseClient } from "@supabase/auth-helpers-react";
 import { Database } from "supabase/db_types";
-
-import { Avatar, IconButton, Chip, Typography, Stack } from "@mui/material";
+import { Avatar, IconButton, Chip, Typography, Stack, Card } from "@mui/material";
 import MaleIcon from "@mui/icons-material/Male";
 import FemaleIcon from "@mui/icons-material/Female";
 import TransgenderIcon from "@mui/icons-material/Transgender";
+import FavoriteIcon from "@mui/icons-material/Favorite";
 import CakeIcon from "@mui/icons-material/Cake";
 import EditIcon from "@mui/icons-material/Edit";
 
 import Navbar from "@/components/public/Navbar";
 import Loading from "@/components/public/Loading";
+import VerifyChip from "@/components/profile/VerifyChip";
 
 import { User } from "@/types/User";
-import { PagePaths } from "enum/pages";
-import { Gender } from "enum/gender";
+import { PAGE_PATHS } from "enum/PAGES";
+import { GENDER } from "enum/GENDER";
```

Comment:
Enum Error

Reply:
`"enum/GENDER"` still error ka.

Reply:
As I stated before, VS code doesnt distinguish upper-lower case. Pls delete the repo and clone again to solve the problem.